src/test/java/io/quarkus/search/app/SearchServiceQuarkiverseTest.java

```diff
@@ -0,0 +1,126 @@
+                .queryParam("q", term)
+                .when().get(GUIDES_SEARCH)
+                .then()
+                .statusCode(200)
+                .extract().body().as(SEARCH_RESULT_SEARCH_HITS);
+    }
+
+    @BeforeAll
+    void setup() {
+        SetupUtil.waitForIndexing(getClass());
+        RestAssured.enableLoggingOfRequestAndResponseIfValidationFails(LogDetail.BODY);
+    }
+
+    @Test
+    void version() {
+        var result = given()
+                .queryParam("q", "orm")
+                .queryParam("version", QuarkusVersions.MAIN)
+                .when().get(GUIDES_SEARCH)
+                .then()
+                .statusCode(200)
+                .extract().body().as(SEARCH_RESULT_SEARCH_HITS);
+        assertThat(result.hits())
+                .isNotEmpty()
+                .allSatisfy(hit -> assertThat(hit).extracting(GuideSearchHit::url, InstanceOfAssertFactories.URI_TYPE)
+                        .asString()
+                        .satisfiesAnyOf(
+                                uri -> assertThat(uri).startsWith("https://quarkus.io/version/"
+                                        + QuarkusVersions.MAIN + "/guides/"),
+                                uri -> assertThat(uri).startsWith("https://docs.quarkiverse.io/")));
+        result = given()
+                .queryParam("q", "orm")
+                .queryParam("version", "main")
+                .when().get(GUIDES_SEARCH)
+                .then()
+                .statusCode(200)
+                .extract().body().as(SEARCH_RESULT_SEARCH_HITS);
+        assertThat(result.hits())
+                .isNotEmpty()
+                .allSatisfy(hit -> assertThat(hit).extracting(GuideSearchHit::url, InstanceOfAssertFactories.URI_TYPE)
+                        .asString()
+                        .satisfiesAnyOf(
+                                uri -> assertThat(uri).startsWith("https://quarkus.io/version/main/guides/"),
+                                uri -> assertThat(uri).startsWith("https://docs.quarkiverse.io/")));
+    }
+
+    @Test
+    void quarkiverse() {
+        var result = given()
+                .queryParam("q", "amazon")
+                .queryParam("version", QuarkusVersions.MAIN)
+                .when().get(GUIDES_SEARCH)
+                .then()
+                .statusCode(200)
+                .extract().body().as(SEARCH_RESULT_SEARCH_HITS);
+        assertThat(result.hits()).extracting(GuideSearchHit::url)
+                .satisfiesOnlyOnce(
+                        uri -> assertThat(uri).asString().contains(GuideRef.QUARKIVERSE_AMAZON_S3.nameBeforeRestRenaming()))
+                .satisfiesOnlyOnce(
+                        uri -> assertThat(uri).asString()
+                                .contains(GuideRef.HIBERNATE_SEARCH_ORM_ELASTICSEARCH.nameBeforeRestRenaming()));
+    }
+
+    @Test
+    @Disabled("Since quarkiverse guides are now fetched directly from their site there is no translation for them available anymore")
```

Comment:
Sounds like a problem, no?

Though I'm not sure why there was a translation in the first place...

Reply:
It's mainly about the Japanese version. There are those few Quarkiverse guides that are mentioned on the quarkus.io/guides, and their titles have been translated. But now, when we index this from the Quarkiverse side, there are no translation files. We probably could try to use the translation files for the main version and hope to find some matches, but I wasn't sure it was worth it. In particular, because the titles are concatenated here + the summary isn't the one from the yaml, so there probably won't be a lot of matches.